src/index.js

```diff
@@ -0,0 +1,31 @@
+import './style.css';
+import refresh from './images/autorenew.svg';
+import add from './images/add.svg';
+import Interact from './module/display.js';
+import { getTodos } from './module/storage.js';
+
+const todos = getTodos();
+const interact = new Interact(todos);
+const todaysTodo = document.querySelector('.todays-todo');
+const addTodo = document.querySelector('.add-todo');
+
+function render() {
+  todaysTodo.innerHTML = `<h4 class="title">Today's To do</h4>
+  <button class="refreshBtn"><img src="${refresh}" alt=""></button>`;
+  addTodo.innerHTML = `<input type="text" class="input-todo required" placeholder="Add to your list...">
+  <button class="addBtn"><img src="${add}" alt=""></button>`;
+}
```

Comment:
It would be better if you use the arrow function `()=>{}` instead of the regular function to make your code clean and simple.

Reply:
Yes, you are right I have to use ES6 syntaxes Thank you. I will correct it as per your suggestions.